app/resources/forms.js.template

```diff
@@ -359,30 +359,24 @@ function validate_fields() {
 function make_form_expandable(expandedSection) {
   var expandedPart = jQuery(expandedSection).find("div.expanded-part");
   var expandButton = jQuery(expandedSection).find("a.expand-button");
+
   expandedPart.hide();
   expandButton.click(function() {
-    if (expandedPart.is(":visible"))
+    if (expandedPart.is(":visible")) {
       expandedPart.slideUp("slow");
-    else
+      if (expandButton.hasClass('expand-button-icon-up')){
+        expandButton.removeClass('expand-button-icon-up').addClass('expand-button-icon-down');
+      }
+    }
```

Comment:
This one should be in the same line as the next "else":
https://google.github.io/styleguide/javascriptguide.xml#Code_formatting